server/gokbg3/grails-app/domain/org/gokb/cred/AllocatedReviewGroup.groovy

```diff
@@ -7,12 +7,13 @@ class AllocatedReviewGroup implements Serializable {
   CuratoryGroup group
   ReviewRequest review
   RefdataValue status
+	AllocatedReviewGroup escalatedFrom
```

Comment:
We will have to handle the case where the escalating ARG defined here is deleted. We may want to delete this ARG as well in that case.

Reply:
There is : 

- https://github.com/openlibraryenvironment/gokb/blob/f767a613ffa8a3b500deea2f3e626e54e6f4634e/server/gokbg3/grails-app/domain/org/gokb/cred/AllocatedReviewGroup.groovy#L46
- https://github.com/openlibraryenvironment/gokb/blob/f767a613ffa8a3b500deea2f3e626e54e6f4634e/server/gokbg3/grails-app/domain/org/gokb/cred/AllocatedReviewGroup.groovy#L50
- https://github.com/openlibraryenvironment/gokb/blob/f767a613ffa8a3b500deea2f3e626e54e6f4634e/server/gokbg3/grails-app/domain/org/gokb/cred/KBComponent.groovy#L1360
- https://github.com/openlibraryenvironment/gokb/blob/f767a613ffa8a3b500deea2f3e626e54e6f4634e/server/gokbg3/grails-app/domain/org/gokb/cred/KBComponent.groovy#L1397

If I understand correctly, in all of these cases the `escalatedFrom` ARG is deleted anyways.